tests/extension_layer_tests.cpp

```diff
@@ -428,18 +428,6 @@ bool VkExtensionLayerTest::AddSurfaceInstanceExtension() {
     }
 #endif
 
-#if defined(VK_USE_PLATFORM_MACOS_MVK)
-    if (!InstanceExtensionSupported(VK_MVK_MACOS_SURFACE_EXTENSION_NAME)) {
-        printf("%s %s extension not supported\n", kSkipPrefix, VK_MVK_MACOS_SURFACE_EXTENSION_NAME);
-        return false;
-    }
-    if (InstanceExtensionSupported(VK_KHR_PORTABILITY_SUBSET_EXTENSION_NAME))
-        instance_extensions_.push_back(VK_KHR_PORTABILITY_SUBSET_EXTENSION_NAME);
-
-    instance_extensions_.push_back(VK_MVK_MACOS_SURFACE_EXTENSION_NAME);
-    bSupport = true;
-#endif
```

Comment:
2 things. We don't test/need this. And this extension isn't deprecated by the VK_EXT_metal_surface extension

Reply:
> And this extension isn't deprecated by the VK_EXT_metal_surface extension

Is deprecated*